Applications/Classification/jni/main.cpp

```diff
@@ -435,7 +435,7 @@ int main(int argc, char *argv[]) {
   NN.init();
   NN.readModel();
 
-  NN.train();
+  int status=NN.train();
```

Comment:
Minor but I think `int status = NN.train();` looks better

Reply:
I don't know how it comes through ci. fixed.